<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FDailyMovement</class>
 <widget class="QWidget" name="FDailyMovement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>866</width>
    <height>43</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Date</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnPrevDate">
     <property name="minimumSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../Resort/res.qrc">
       <normaloff>:/images/left.png</normaloff>:/images/left.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="EDateEdit" name="deStart"/>
   </item>
   <item>
    <widget class="QPushButton" name="btnNextDate">
     <property name="minimumSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../Resort/res.qrc">
       <normaloff>:/images/right.png</normaloff>:/images/right.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Sale type</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="EQLineEdit" name="leSale">
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="EnableHiddenText" stdset="0">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chCanceled">
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>Canceled</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>142</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EQLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">eqlineedit.h</header>
   <slots>
    <signal>customButtonClicked(bool)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>EDateEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">edateedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../Resort/res.qrc"/>
 </resources>
 <connections/>
</ui>
